standard/standard/clause_7_normative_text.adoc

```diff
@@ -397,700 +348,669 @@ The Graph Target supports a network of object relative poses. The graph is a dir
-<s|ID |<<req_regular_series_header/structure,/req/regular_series/header/sdu>>
-2+<|A header property SHALL be implemented as a SeriesHeader structure.
-|===
-
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|GP-Duration-SDU
-<s|ID |<<req_time_geopose_duration,/req/time/geopose_duration/SDU>>
-<s|Requirement |An interposeDuration property SHALL be implemented as a GeoPoseDuration.
-|===
-
-
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|FS-Authority-SDU
-<s|ID |<<req_sdu_frame_specification_authority,/req/frame_specification/authority/sdu>>
-2+<|The frame_specification.authority attribute SHALL contain a string uniquely specifying a source of reference frame specifications.
-|===
-
-
-[width="100%",cols="4,6"]
-|===
-<s|*Requirement s| FS-ID-SDU 
-<s|ID |<<req_sdu_frame_specification_id,/req/frame_specification/id/sdu>>
-2+<|The frame_specification.ID attribute SHALL be a string uniquely defining a frame within the authority.
-|===
-
-
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|FS-Parameters-SDU
-<s|ID |<<req_sdu_frame_specification_parameter,/req/frame_specification/parameters/sdu>>
-2+<|The frame_specification.parameter attribute SHALL contain all parameters needed for the corresponding authority and ID.
-|===
-
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|S-Trailer-SDU
-<s|ID |<<req_series_regular_trailer,/req/series/regular/trailer/sdu>>
-2+<|A trailer property SHALL be implemented as a SeriesTrailer structure.
-|===
-
-<<<
+image::diagram/Regular-Series_SDU.PNG[]
+
+[[req_regular_series_sdu]]
+[requirements_class]
+====
+[%metadata]
+label:: /req/regular_series/sdu
+subject:: Implementation of Logical Model
+description:: Regular Series Logical Model SDU
+inherit:: /req/global
+
+requirement:: /req/regular_series/header/sdu
+requirement:: /req/time/geopose_duration/SDU
+requirement:: /req/frame_specification/authority/sdu
+requirement:: /req/frame_specification/id/sdu
+requirement:: /req/frame_specification/parameters/sdu
+requirement:: /req/series/regular/trailer/sdu
+====
+
+[requirement]
+.S-Header-SDU
+====
+[%metadata]
+label:: /req/regular_series/header/sdu
+description:: A header property SHALL be implemented as a SeriesHeader structure.
+====
+
+[requirement]
+.GP-Duration-SDU
+====
+[%metadata]
+label:: /req/time/geopose_duration/SDU
+description:: An interposeDuration property SHALL be implemented as a GeoPoseDuration.
+====
+
+[requirement]
+.FS-Authority-SDU
+====
+[%metadata]
+label:: /req/frame_specification/authority/sdu
+description:: The frame_specification.authority attribute SHALL contain a string uniquely specifying a source of reference frame specifications.
+====
+
+[requirement]
+.FS-ID-SDU
+====
+[%metadata]
+label:: /req/frame_specification/id/sdu
+description:: The frame_specification.ID attribute SHALL be a string uniquely defining a frame within the authority.
+====
+
+[requirement]
+.FS-Parameters-SDU
+====
+[%metadata]
+label:: /req/frame_specification/parameters/sdu
```

Comment:
This Requirement is duplicated.

Reply:
All duplicated requirements must be removed. Only keep the first instance.